sycl/unittests/pi/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+add_llvm_unittest(PiTests
+  main.cpp
+  piPlatformTest.cpp
+  piProgramTest.cpp
+)
+
+add_dependencies(PiTests sycl)
+target_link_libraries(PiTests PRIVATE sycl LLVMTestingSupport)
+target_compile_options(PiTests PRIVATE -fexceptions)
```

Comment:
Can you please wrap this with if (NOT MSVC)?

Reply:
Yes, no problem!